plugins/modem-manager/fu-mm-device.c

```diff
@@ -116,6 +123,7 @@ validate_firmware_update_method (MMModemFirmwareUpdateMethod methods, GError **e
 	static const MMModemFirmwareUpdateMethod supported_combinations[] = {
 		MM_MODEM_FIRMWARE_UPDATE_METHOD_FASTBOOT,
 		MM_MODEM_FIRMWARE_UPDATE_METHOD_QMI_PDC | MM_MODEM_FIRMWARE_UPDATE_METHOD_FASTBOOT,
+		MM_MODEM_FIRMWARE_UPDATE_METHOD_MBIM_QDU,
```

Comment:
This update method does not exist in ModemManager. This pull request should have been sent for review after the corresponding update method is defined and used in ModemManager, otherwise, is this pull request only usable with some unknown custom version of MM?

Could you please push the MM changes first, we get them merged in MM, we then bump the MM dev version including the changes (e.g. to 1.17.1 or whatever) and then we build-depend on that version for this integration in fwupd?

Reply:
Yes, we definitely need the MM changes merged first -- good catch.

Reply:
> Could you please push the MM changes first, we get them merged in MM, we then bump the MM dev version including the changes (e.g. to 1.17.1 or whatever) and then we build-depend on that version for this integration in fwupd?

Sure, the qdu related parts in MM will be pushed soon, if no problems, please merged it in MM, thanks.

Reply:
I have now merged to git master in MM, and pushed a new version bump to 1.17.1, so, can you rework your changes so that the MBIM QDU support is included in between `#if MM_CHECK_VERSION (1,17,1) ...   #endif` checks?